bofire/benchmarks/single.py

```diff
@@ -494,3 +495,140 @@ def __init__(self, **kwargs):
 
     def get_optima(self):
         raise NotImplementedError()
+
+
+class Multinormalpdfs(Benchmark):
+    """The sum of probability density functions of multivariate Gaussian distributions
+
+    A virtual experiment in this benchmark class is evaluated by summing one or more
+    multivariate normal PDFs:
+        y_expt = sum_i {const_i * exp(-0.5 * (x - mu_i)^T Sigma_i^-1 (x - mu_i))}
+    where mu_i is the i-th mean vector and Sigma_i is the i-th covariance matrix.
+
+    This allows us to control
+        * the possible presence of multiple optima by choosing the number of Gaussians
+        * the position of the optimum/optima by setting the mean(s)
+        * the axis-alignment of each elliptical dip in the objective landscape by modifying the
+          covariance matrix/matrices
+        * the presence of non-influential variables by setting the corresponding elements in the
+          covariance matrix to zero or a large number
+    """
+
+    def __init__(
+        self,
+        dim: int = 5,
+        n_gaussians: int = 1,
+        stdev: float = 0.4,
+        eigscale: float = 0.5,
+        opt_on_boundary: bool = False,
+        N_unimportant_inputs: int = 2,
+        means: Optional[list] = None,
+        covmats: Optional[np.ndarray] = None,
```

Comment:
It would be nice to also add a `seed` argument of type `seed: Optional[int] = None`, to be able to reproduce randomly generated response surfaces.

Reply:
We should then also test in the test that it produces the same numbers when a seed is set.